### PR TITLE
task: remove tc agreement for returning users PE-734

### PR DESCRIPTION
**Pull request by jdaev**

### CODE REVIEW

File: lib/pages/profile_auth/components/profile_auth_add_screen.dart [hunk 1]
```diff
@@ -89,42 +89,44 @@ class ProfileAuthAddScreen extends StatelessWidget {
...
                               ),
-                            ),
-                          ],
-                        ),
+                            ],
+                          ),
+                        ],
                         const SizedBox(height: 32),
                         SizedBox(
                           width: double.infinity,
```
Comment (matibat) [inline lib/pages/profile_auth/components/profile_auth_add_screen.dart:132]: I'm not sure to what is `SizedBox` related to, but is `double.infinity` safe here?

File: lib/pages/profile_auth/components/profile_auth_add_screen.dart [hunk 2]
```diff
@@ -72,7 +72,7 @@ class ProfileAuthAddScreen extends StatelessWidget {
                               context.read<ProfileAddCubit>().submit(),
                           validationMessages: (_) => kValidationMessages,
                         ),
-                        if (!state.isExistingUser) ...{
+                        if (!state.isExistingUser) ...[
```
Comment (matibat) [inline lib/pages/profile_auth/components/profile_auth_add_screen.dart:75]: I'd guess this is an inline conditional, right? In the else case there's nothing to push to this `List` here I assume

**Review by matibat — COMMENTED** [review]
Left a few questions there

File: lib/pages/profile_auth/components/profile_auth_add_screen.dart (hunk 1 above)
Comment (jdaev) [reply lib/pages/profile_auth/components/profile_auth_add_screen.dart:132]: Yes. It's the value typically used for denoting MAX in flutter

File: lib/pages/profile_auth/components/profile_auth_add_screen.dart (hunk 2 above)
Comment (jdaev) [reply lib/pages/profile_auth/components/profile_auth_add_screen.dart:75]: It's a language feature.
https://dart.dev/guides/language/language-tour#collection-operators

**Review by fedellen — APPROVED** [review]
👍

**Pull request merged**